core/modules/config/tests/config.test

```diff
@@ -539,6 +539,34 @@ class ConfigurationUITest extends BackdropWebTestCase {
     $this->assert(!empty($staging_files), 'Staging directory is not empty.');
   }
 
+  /**
+   * Test comparing configs whose elements are in different order.
+   */
+  public function testConfigOrderChange() {
```

Comment:
This doesn't have to be public, it could be protected.

Reply:
I can, but all the other test methods are all public in this class. Why would we want to change that?

Reply:
I know, method visibility in functional tests is a total mess throughout core. :wink: 

It doesn't change anything substantially, but I belief, they should all be "protected". At least, newly created ones should be. I'll leave it up to you to change, or keep the current visibility. It's really not relevant for the actual bugfix here.